js/src/lib/components/InferenceWidget/shared/WidgetExample.ts

```diff
@@ -0,0 +1,94 @@
+import type { TableData } from "./types";
+
+//#region outputs
+export type WidgetExampleOutputLabels = Array<{ label: string; score: number; }>;
+export interface WidgetExampleOutputAnswerScore {
+	answer: string;
+	score:  number;
+}
+export interface WidgetExampleOutputText {
+	text: string;
+}
+export interface WidgetExampleOutputUrl {
+	url: string;
+}
+//#endregion
+
+export interface WidgetExampleBase {
+	example_title?: string;
+	group?:         string;
+}
+
+export interface WidgetExampleTextInputLabelsOutput extends WidgetExampleBase {
```

Comment:
nit: naming wise we can remove the ending `Output`? Instead of `WidgetExampleTextInputLabelsOutput`, just `WidgetExampleTextInputLabels`